### PR TITLE
Submit bunch of translation patches.

### PR DESCRIPTION
**Pull request by jondong**
After this one, we are going to submit the translation directly into crosswalk-project/crosswalk-website repo.

### CODE REVIEW

**Comment by jondong** [comment]
@bspencer PTAL, thanks

File: public/documentation/manifest/_data.json [hunk 1]
```diff
@@ -1 +1,82 @@
...
  },
  "other_fields": {
    "title": "Other fields"
  }
}
\ No newline at end of file
+{
+  "using_the_manifest": {
+     "title": "Using the manifest",
+     "language": "en"
```
Comment (bspencer) [inline public/documentation/manifest/_data.json]: Are these supposed to have "language":"en"?  Is this from an old commit?

File: public/documentation/manifest/_data.json [hunk 2]
```diff
@@ -1 +1,82 @@
...
+  "other_fields": {
+     "title": "Other fields",
+     "language": "en"
+  },
+  "using_the_manifest_zh": {
+     "title": "使用manifest文件",
+     "language": "zh"
+  },
+  "entry_points_zh": 
+     "title": "应用入口点",
```
Comment (lisa0314) [inline public/documentation/manifest/_data.json]: missing ' { ' after "entry_points_zh"

File: public/documentation/manifest/_data.json [hunk 3]
```diff
@@ -1 +1,82 @@
...
+  },
+  "other_fields": {
+     "title": "Other fields",
+     "language": "en"
+  },
+  "using_the_manifest_zh": {
+     "title": "使用manifest文件",
+     "language": "zh"
+  },
+  "entry_points_zh":
```
Comment (bspencer) [inline public/documentation/manifest/_data.json]: Should have "{" at the end of this line

**Pull request merged**